doc/source/security/innodb-encryption.rst

```diff
@@ -0,0 +1,405 @@
+.. _innodb_encryption:
+
+========================================================
+InnoDB Encryption
+========================================================
+
+|Percona Server| provides tools to protect data with tablespace encryption. These tools are a part of the data-at-rest encryption feature.
+
+InnoDB System Tablespace Encryption
+================================================================================
+
+:Availabiliity: This feature is **Experimental** quality
+
+The InnoDB system tablespace is encrypted with the master key encryption. If the variable :variable:`innodb_sys_tablespace_encrypt` is set to ON and the
+server is started with the ``--bootstrap`` option, you may create an encrypted table as follows:
+
+.. code-block:: guess
+
+   mysql> CREATE TABLE ... TABLESPACE=innodb_system ENCRYPTION='Y'
+
+.. note::
+
+   Any tables in the System tablespace before this command is issued cannot be encrypted.
+
+After a system tablespace is encrypted, you cannot convert the tablespace to unencrypted and an unencrypted system tablespace cannot be converted to encrypted. A new instance should be created and the user tables are transferred to the instance.
```

Comment:
the system tablespace.

this paragraph also seems to be redundant with another file?

Reply:
Doc removed.